site/content/en/docs/tasks/integrate_a_custom_job.md

```diff
@@ -0,0 +1,58 @@
+---
+title: "Integrate a custom Job"
+date: 2023-07-25
+weight: 8
+description: >
+  Integrate a custom Job with Kueue.
+---
+# Integrate a custom Job with Kueue
+
+This page shows how to integrate a custom Job with Kueue. 
+
+## What you need to specify
+
+1. Configuration
+2. Controller
+3. Webhook
+4. Test files
+5. Adjust build system
+
+## Configure your custom Job
+
+Add your framework name to `.integrations.frameworks` in [controller_manager_config.yaml](https://github.com/kubernetes-sigs/kueue/blob/main/config/components/manager/controller_manager_config.yaml)
+
+## Build controller
+
+Provide your code of the controller under a dedicated directory. This directory need to under the `./pkg/controller/jobs/` folder.
+
+1. Register a new framework
```

Comment:
I added this because I wanted to mention that `jobframework.RegisterIntegration` is required to provide.

Reply:
> Not sure what is meant here, I think the framework registration is what we do under `Add your framework name to `.integrations.frameworks` in [controller_manager_config.yaml](https://github.com/kubernetes-sigs/kueue/blob/main/config/components/manager/controller_manager_config.yaml)`. I would suggest to remove this point.

I added this because I wanted to mention that `jobframework.RegisterIntegration` is required to provide in `init()` in controller file.

Reply:
sgtm